child_compassion/models/project_lifecycle_event.py

```diff
@@ -112,7 +112,7 @@ def create(self, vals):
         if event.type == 'Reactivation':
             if fund_suspended:
                 project.reactivate_project()
-            if hold_gifts and not event.hold_gifts():
+            if hold_gifts and not event.hold_gifts:
```

Comment:
This test is nonsense, it will always be false. You should have renamed method hold_gifts and keep the call here.